auto_backup/README.rst

```diff
@@ -0,0 +1,108 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Database Auto-Backup
+====================
+
+A tool for all your back-ups, internal and external!
+
+Installation
+============
+
+Before installing this module, you need to execute::
+
+    pip install pysftp
+
+Configuration
+=============
+
+Go to *Settings -> Configuration -> Configure Backup* to
+create your configurations for each database that you needed
+to backups.
+
+Usage
+=====
+
```

Comment:
We ran into an issue where backup execution was dying when running inside of Odoo due to the timeouts configured on the instance. We had to make a small script to run this system outside of the main Odoo instance to circumvent this, which is described [in this blog post](https://blog.laslabs.com/2016/10/running-python-scripts-within-odoos-environment/).

If this is something other people have run into as well, maybe transferring some of that info to this ReadMe would be of use?